changelogs/fragments/bump_215.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Could you also add release summary , ref: https://github.com/rohitthakur2590/arista.eos/blob/5e84fc61ac464894be5c2567bc637f7c2d5f497e/changelogs/fragments/bump215.yaml